hail/python/test/hailtop/batch/test_batch_service_backend.py

```diff
@@ -507,6 +507,21 @@ def test_specify_job_region(service_backend: ServiceBackend):
     assert res_status['state'] == 'success', str((res_status, res.debug_info()))
 
 
+def test_use_default_region_when_not_specifying_any_regions(service_backend: ServiceBackend):
+    b = batch(service_backend)
+    j = b.new_job('region')
+    default_region = service_backend.default_region()
+    j.regions()
```

Comment:
This needs an argument. What was the intention? Don't you want to test when the region isn't set explicitly?

Reply:
I wanted to make sure it was set to None in case the default was set anywhere in the test suite. I'm just passing `None`.